src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -816,6 +816,17 @@ static Library()
                 BuiltinFunctionsCore.IsBlankOrErrorOptionSetValue,
                 NoErrorHandling(IsBlankOrError)
             },
+            {
+                BuiltinFunctionsCore.IsEmpty,
+                StandardErrorHandling<FormulaValue>(
+                    BuiltinFunctionsCore.IsEmpty.Name,
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: DoNotReplaceBlank,
+                    checkRuntimeTypes: DeferRuntimeTypeChecking,
+                    checkRuntimeValues: DeferRuntimeValueChecking,
+                    returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
```

Comment:
>AlwaysEvaluateAndReturnResult [](http://example.com/codeflow?start=51&length=29)

Since we are returning FALSE when arg[0] is Blank(), does it make sense change this to ReturnFalseIfAnyArgIsBlank?

Reply:
Will update it after we decide with the team what should be the value of something like `IsEmpty(If(1<0,[1,2,3]))`